specs/lib/createGltfSpec.js

```diff
@@ -76,208 +113,20 @@ describe('createGltf', function() {
     it('multiple nodes, meshes, and primitives', function() {
         var gltf = createGltf(groupObjData, defaultOptions);
 
-        expect(Object.keys(gltf.materials).length).toBe(3);
-        expect(Object.keys(gltf.nodes).length).toBe(1);
-        expect(Object.keys(gltf.meshes).length).toBe(3);
+        expect(gltf.materials.length).toBe(3);
+        expect(gltf.nodes.length).toBe(4);
+        expect(gltf.nodes[0].mesh).toBeUndefined();
```

Comment:
I guess for "correctness" we would want this to check the node(s) in `gltf.scenes[gltf.scene]`.